### PR TITLE
CB-1392 add plain name for database nodes

### PR DESCRIPTION
**Pull request by yagudin10**

### CODE REVIEW

File: webapp/packages/plugin-navigation-tree/src/NodesManager/NavNodeContextMenuService.ts [hunk 1]
```diff
@@ -123,7 +123,7 @@ export class NavNodeContextMenuService extends Bootstrap {
       },
       handler: async (context, action) => {
         const node = context.get(DATA_CONTEXT_NAV_NODE);
-        const name = getNodeDisplayName(node);
+        const name = node.plainName ?? getNodeDisplayName(node);
```
Comment (Wroud) [inline webapp/packages/plugin-navigation-tree/src/NodesManager/NavNodeContextMenuService.ts]: please create function `getNodePlainName(node)`

File: webapp/packages/plugin-navigation-tree/src/NavigationTree/ElementsTree/NavigationTreeNode/NavigationNode/NavigationNodeControl.tsx [hunk 2]
```diff
@@ -146,7 +146,7 @@ export const NavigationNodeControl: NavTreeControlComponent = observer<NavTreeCo
         <TreeNodeName title={title} {...use({ editing })}>
           <Loader suspense inline fullSize>
             {editing ? (
-              <NavigationNodeEditorLoader name={name} disabled={saving} onSave={editingState.save} onClose={editingState.cancel} />
+              <NavigationNodeEditorLoader name={node.plainName ?? name} disabled={saving} onSave={editingState.save} onClose={editingState.cancel} />
```
Comment (Wroud) [inline webapp/packages/plugin-navigation-tree/src/NavigationTree/ElementsTree/NavigationTreeNode/NavigationNode/NavigationNodeControl.tsx]: use here `getNodePlainName(node)`

File: server/bundles/io.cloudbeaver.server/schema/service.navigator.graphqls [hunk 3]
```diff
@@ -59,7 +59,9 @@ type NavigatorNodeInfo {
     # Node human readable name
     name: String
     #Node full name
-    fullName: String
+    fullName: String @deprecated
+    #Node plain name
+    plainName: String
```
Comment (Wroud) [inline server/bundles/io.cloudbeaver.server/schema/service.navigator.graphqls]: https://github.com/dbeaver/pro/wiki/GraphQL-Schema-Guidelines
please specify deprecation message: version when deprecated, comment what to use instead

**Review by Wroud — CHANGES_REQUESTED** [review]

File: server/bundles/io.cloudbeaver.server/schema/service.navigator.graphqls (hunk 3 above)
Comment (yagudin10) [reply server/bundles/io.cloudbeaver.server/schema/service.navigator.graphqls]: @Wroud could you explain what is used instead of fullName?

Comment (Wroud) [reply server/bundles/io.cloudbeaver.server/schema/service.navigator.graphqls]: i don't know but if field deprecated you need to describe in favor of what
i suppose it's `name`

**Review by Wroud — APPROVED** [review]

**Pull request merged**